cmd/tikv-ctl/src/cmd.rs

```diff
@@ -627,6 +630,62 @@ pub enum Cmd {
         /// hex end key
         end: String,
     },
+    CompactLogBackup {
```

Comment:
I personally think it's better be placed in BR.

Reply:
the deployment is a standalone command in a one-shot pod to scan logs and convert them to sst files. It run offline from normal br process.